README.md

```diff
@@ -145,6 +145,10 @@ Check out [node-pg-pool](https://github.com/brianc/node-pg-pool) and [node postg
       <td>Boolean</td>
       <td>Whether to try SSL/TLS to connect to server</td>
     </tr>
+    <tr>
+      <td>defaultSort</td>
+      <td>boolean / string</td>
```

Comment:
Nitpick: `boolean / string` -> `Boolean/String`

Reply:
Thanks, corrected it.